test/helpers/test-data.js

```diff
@@ -20,7 +20,7 @@ export const VALID_MEASUREMENT = {
   provider_address: '/dns4/production-ipfs-peer.pinata.cloud/tcp/3000/ws/p2p/Qma8ddFEQWEU8ijWvdxXm3nxU7oHsRtCykAaVz8WUYhiKn',
   providerId: 'PROVIDERID',
   spark_version: '1.10.4',
-  protocol: 'bitswap',
+  protocol: 'http',
```

Comment:
what is this change for?

Reply:
It's a bit unrelated. I thought this change would reduce the number of tests I need to fix as part of this PR, but after I reverted the change, all tests are passing, so it's not necessary at all.

Reverted.